### PR TITLE
[CP4BA] Add custom database for BAW

### PR DESCRIPTION
**Pull request by P-147**
Hello cpd team! 

Today, I'm adding a small improvement for BAW (Business Automation Workflow) in the CP4BA 😄
This is something I had to configure manually multiple times for client demos, and would like to automate in the future.

**TL;DR:**
This patch add the creation of a new database `baw_external` on the Postgres instance, and register it on the Liberty server running BAW (Authoring).  
This allow workflow designers to reference this new db from BAW, and perform SQL operations using the 'System Data' Toolkit.

**List of Changes:**
- Add ansible create / delete tasks for `baw_external` database
- Add example table `user_db` with sample data in the `baw_external` db
- Renamed Secret template `bawaut-custom-xml-secret.yaml.j2` ‎to `bawaut-lombardi-custom-xml-secret.yaml.j2` for clarity
- Add Secret template `bawaut-liberty-custom-xml-secret.yaml.j2` with `baw_external` connection informations and register it on the CP4BA operator instance
- Add documentation in `cp4ba-usage` on how to use the `baw_external` db connection in BAW

Let me know if I need to change something :)

### CODE REVIEW

File: automation-roles/50-install-cloud-pak/cp4ba/cp4ba-core/tasks/predeploy/bawaut.yml [hunk 1]
```diff
@@ -30,13 +30,26 @@
 # Based on https://www.ibm.com/docs/en/cloud-paks/cp-biz-automation/latest?topic=services-optional-customizing-runtime-server-properties
 # Based on https://www.ibm.com/docs/en/baw/20.x?topic=work-configuring-email-notifications
 # Based on https://www.ibm.com/docs/en/baw/20.x?topic=configuration-creating-100customxml-file
-- name: Prepare yaml file for BAWAUT Secret
+- name: Prepare yaml file for BAWAUT Lombardi Custom100.xml Secret
```
Comment (jandusek4) [inline automation-roles/50-install-cloud-pak/cp4ba/cp4ba-core/tasks/predeploy/bawaut.yml:33]: Custom100.xml to 100Custom.xml

File: automation-roles/50-install-cloud-pak/cp4ba/cp4ba-core/tasks/predeploy/bawaut.yml [hunk 2]
```diff
@@ -30,13 +30,26 @@
...
   ansible.builtin.template:
-    src: bawaut/bawaut-custom-xml-secret.yaml.j2
-    dest: "{{ cp4ba_output_directory }}/bawaut-custom-xml-secret.yaml"
+    src: bawaut/bawaut-lombardi-custom-xml-secret.yaml.j2
+    dest: "{{ cp4ba_output_directory }}/bawaut-lombardi-custom-xml-secret.yaml"
     mode: u+rwx
 
-- name: Add BAWAUT Secret
+- name: Add BAWAUT Lombardi Custom100.xml Secret
```
Comment (jandusek4) [inline automation-roles/50-install-cloud-pak/cp4ba/cp4ba-core/tasks/predeploy/bawaut.yml:39]: Custom100.xml to 100Custom.xml

**Review by jandusek4 — CHANGES_REQUESTED** [review]
Custom100.xml to 100Custom.xml

File: automation-roles/50-install-cloud-pak/cp4ba/cp4ba-core/tasks/predeploy/bawaut.yml (hunk 1 above)
Comment (P-147) [reply automation-roles/50-install-cloud-pak/cp4ba/cp4ba-core/tasks/predeploy/bawaut.yml]: Fixed in next commit  (391bb97)

File: automation-roles/50-install-cloud-pak/cp4ba/cp4ba-core/tasks/predeploy/bawaut.yml (hunk 2 above)
Comment (P-147) [reply automation-roles/50-install-cloud-pak/cp4ba/cp4ba-core/tasks/predeploy/bawaut.yml]: Fixed in next commit  (391bb97)

**Review by jandusek4 — APPROVED** [review]
Now its all good!

**Pull request merged**